python_modules/libraries/dagster-k8s/dagster_k8s/client.py

```diff
@@ -736,6 +736,7 @@ def get_pod_debug_info(
         pod_name,
         namespace,
         pod: Optional[kubernetes.client.V1Pod] = None,  # the already fetched pod
+        skip_logs: bool = False,
```

Comment:
nit I find double-negatives hard to reason about - consder "include_container_logs" = True instead?

Reply:
fair, i will change it